src/App/styles.ts

```diff
@@ -0,0 +1,17 @@
+import { css } from 'emotion';
+import colors from '../utils/styles/colors';
+
+export const alertError = css`
+  background-color: ${colors.red} !important;
```

Comment:
I don't use "!important;" anymore..I think when you need to use it...something is wrong..

Reply:
Completely agree with you on this. 
This was a copy from the scss files I didn't want to mix up with actually refactoring styles. I think this most likely will be canned with Material-UI?

Reply:
In case we need this, we can surround in `&&` block to repeat the css class, making it important without explicitly writing the taboo word (!important)
```css
&& {
  background-color: ${colors.red};
}
```

Reply:
Ouch, I saw `!important` in more files, so we can keep it for a while and then remove, it isn't in the scope of the PR (great work, I'm reviewing :wink:)

Reply:
Agree, I dislike important, which sounds funny. But, it was there already so, cool for now. Legacy stuff.